merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/services/LocalMissionModelService.java

```diff
@@ -246,7 +247,7 @@ private MissionModelFacade.Unconfigured<?> loadUnconfiguredMissionModel(final St
   private MissionModelFacade loadConfiguredMissionModel(final String missionModelId)
   throws NoSuchMissionModelException, MissionModelLoadException
   {
-    return loadConfiguredMissionModel(missionModelId, SerializedValue.of(Map.of()));
+    return loadConfiguredMissionModel(missionModelId, Instant.EPOCH, SerializedValue.of(Map.of()));
```

Comment:
🔶 I worry about this "bogus" value. I'm imagining a mission model using it to instantiate spice-related values, and blowing up because the kernels don't cover January 1st, 1970.

Reply:
That's a really good point. This is the one area I'm the most unsure about. Let me think about this for another minute.

Reply:
The only 2 alternatives I can think of are:
1. Have the mission model define a sane default `Instant` plan start time (through an annotation) denoting a time that should always "work" with the mission model.
2. Require plan ID whenever loading a "configured mission model". This change would bubble up to our GQL layer.

Reply:
I agree with your assessment. I'm not sure which of the two options is preferable.

Reply:
I think option 2 above is sort of unavoidable, at least with how we have our mission model loading implemented.

Reply:
Another solution would be to represent "bogus" at the type-level. For example, by using an `Optional<Instant>` instead of an `Instant`. This implementation detail would leak to the mission modeler since they would conditionally use the time depending on if it's empty or not.

If an `Optional<Instant>` is frowned upon we could use a `sealed` type called `PlanStart` which could be `Empty` or `Provided` instead.

Reply:
Basically with this approach the mission modeler is aware of when a bogus start time is being used within the top-level mission model constructor. Not the best solution but may be good enough until we figure out how to refactor our mission model loading (which is a sizable amount of work).

Reply:
I'm actually pretty convinced this is a decent solution until we redesign our mission model loading.

With this PR a `PlanStart` parameter to top-level mission model c-tor is completely optional, so no backwards compat. is being broken and it's not a big deal if mission modelers have to be aware of when `PlanStart` is `instanceof PlanStart.Empty` for a few releases.

Reply:
Pushed change to add an `UNTRUE_PLAN_START` env. var. This is threaded from the app driver to the mission model service.